netscaler/config_test.go

```diff
@@ -17,12 +17,14 @@ package netscaler
 
 import (
 	"fmt"
-	"github.com/chiradeep/go-nitro/config/basic"
-	"github.com/chiradeep/go-nitro/config/lb"
 	"log"
 	"math/rand"
 	"testing"
 	"time"
+
+	"github.com/chiradeep/go-nitro/config/basic"
+	"github.com/chiradeep/go-nitro/config/lb"
+	"github.com/go-nitro/config/ns"
```

Comment:
Build still fails here. This simply cannot be a valid import. It has to have my repository in it.

Reply:
I don't know why it must be failing, only reason i could think of is the path of running it. This is the path from where I am running the code from.
```
$ pwd
~/go/src/github.com/go-nitro
```

Reply:
@chiradeep: I was able to build the code by running the code from this path too: `~/go/src/github.com/chiradeep/go-nitro`.  I was going through your PR again and I don't see the config file `config/ns/nsacls.go` in it. https://github.com/chiradeep/go-nitro/commit/42a5d8877682a75ef562bbd4dc50f1fcc7b80b40

Do you mind adding the config file to the library `chiradeep/go-nitro/config/ns/nsacls.go` . Add this:

```
package ns

type Nsacls struct{}
```

Adding this should solve the import problem of `github.com/chiradeep/go-nitro/config/ns` inside `config_test.go`.

Reply:
OK, will try.